tests/test_pytest_icdiff.py

```diff
@@ -229,6 +229,21 @@ def test_one():
     assert comparison_line.count("hell") < 15
 
 
+def test_mutliline_strings_have_no_escaped_newlines(testdir):
+    testdir.makepyfile(
+        r"""
+        def test_one():
+            one = "a\nb\nc\nd"
+            two = "a\nb\nc\ndd"
+            assert one == two"""
+    )
+    output = testdir.runpytest("-vv", "--color=yes").stdout.str()
+    print(repr(output))
+    assert "a" + " " * 36 in output
+    assert "b" + " " * 36 in output
+    assert "c" + " " * 36 in output
```

Comment:
i don't quite understand what these asserts are doing?

Reply:
That's to make sure that there're actually empty spaces instead of new lines after last characters on the line
<img width="1364" alt="image" src="https://github.com/hjwp/pytest-icdiff/assets/6176998/1f08eac5-7115-4b58-b780-1809dc33ecbe">

before it looked like this:
<img width="869" alt="image" src="https://github.com/hjwp/pytest-icdiff/assets/6176998/d55cae94-e885-434b-9574-1b8a2d38c36a">
so this test would fail

36 should be slightly above the actual size of these columns, with some margins in case icdiff implementation changes a bit.